CMakeLists.txt

```diff
@@ -28,42 +28,56 @@ cmake_minimum_required(VERSION 3.2)
 
 project(VulkanHppGenerator)
 
-file(TO_NATIVE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/Vulkan-Docs/src/spec/vk.xml vk_spec)
-string(REPLACE "\\" "\\\\" vk_spec ${vk_spec})
-add_definitions(-DVK_SPEC="${vk_spec}")
+# If you wish to target a specific version of the spec, override this value like so 
+# cmake -DVK_SPEC_URL:STRING=https://example.com/some/other/url/vk.xml
+set(VK_SPEC_URL "https://raw.githubusercontent.com/KhronosGroup/Vulkan-Docs/1.0/src/spec/vk.xml" CACHE STRING "URL from which to fetch the Vulkan Spec XML")
 
-file(TO_NATIVE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/vulkan/vulkan.hpp vulkan_hpp)
-string(REPLACE "\\" "\\\\" vulkan_hpp ${vulkan_hpp})
-add_definitions(-DVULKAN_HPP="${vulkan_hpp}")
-
-set(HEADERS
-)
+# Download the request URL specification
+file(DOWNLOAD ${VK_SPEC_URL} ${CMAKE_CURRENT_BINARY_DIR}/vk.xml )

```

Comment:
Why not download the vk.xml spec with a version number in the file name so it's clear what version you are getting/building?